index.html

```diff
@@ -0,0 +1,102 @@
+        <p id="sec-para">2022.11.16(Tue) ~ 25(Thur)</p>
+        <p id="third-para">@International Conference Centre.</p>
+    </section >
+    <section class="line-programm">
+        <div class="programm">
+            <h3 class="head-text">Main Program</h3>
+            <div class="under-line"></div>
+        </div>
+        <div class="programm-1">
+            <div class="main-programm">
+                <img src="assets/computer-chat.png"/>
+                <h4 id="card-theme">Marketing</h4>
+                <p>Meet Associate professor from the University of Michigan discussing the recent tech trends</p>
+            </div>
+            <div class="main-programm">
+                <img src="assets/external-computer-network-communications-indigo-line-kalash.png"/>
+                <h4 id="card-theme">Security</h4>
+                <p>Meet Associate professor from the University of Michigan discussing the recent tech trends</p>
+            </div>
+            <div class="main-programm">
+                <img src="assets/external-computer-network-communications-indigo-line-kalash-2.png"/>
+                <h4 id="card-theme">Hacking</h4>
+                <p>Meet Associate professor from the University of Michigan discussing the recent tech trends</p>
+            </div>
+            
+            <div class="main-programm">
+                <img src="assets/external-computer-network-communications-indigo-line-kalash.png"/>
+                <h4 id="card-theme">Networks</h4>
+                <p>Meet Associate professor from the University of Michigan discussing the recent tech trends</p>
+            </div>
+            
+            <div class="main-programm">
+                <img src="assets/google-compute-engine.png"/>
+                <h4 id="card-theme">Innovations</h4>
+                <p>Meet Associate professor from the University of Michigan discussing the recent tech trends</p>
+            </div>
+            
+        </div>
+        <div class="assets/external-computer-network-communications-indigo-line-kalash.png">
+            <p class="head-text">Join the Tech Program 2022</p>
+        </div>
+    </section>
+        <section class="featured">
+            <h3>Featured Speakers</h3>
+            <div class="under-line"></div>
+            <div class="main-speakers"></div>
+            <button type="button" class="show-speakers">show more</button>
+        </section>
+        <section class="partners">
+            <h4 class="logo-content">Partners</h4>
+            <div class="under-line"></div>
+            <div class="partner-details">
+                <div id="partner-name">Mozilla</div>
+                <div id="partner-name">Google</div>
+                <div id="partner-name">NAVER</div>
+                <div id="partner-name">Slack</div>
+                <div id="partner-name">IBM</div>
+            </div>
+        </section>
```

Comment:
No need to have several sections in the project. The project should be divided into header, three sections, and the footer. The sections should include, the top section for the information of the summit, the middle section for the main program, and the bottom section for the speakers. You can use article or divs for the other containers within each section.